deps.toml

```diff
@@ -113,14 +125,19 @@ debezium-mongodb = { module = "io.debezium:debezium-connector-mongodb", version.
 debezium-mysql = { module = "io.debezium:debezium-connector-mysql", version.ref = "debezium"}
 debezium-postgres = { module = "io.debezium:debezium-connector-postgres", version.ref = "debezium"}
 
+
 [bundles]
 apache = ["apache-commons", "apache-commons-lang"]
 datadog = ["datadog-trace-api", "datadog-trace-ot"]
 jackson = ["jackson-databind", "jackson-annotations", "jackson-dataformat", "jackson-datatype"]
 junit = ["junit-jupiter-api", "junit-jupiter-params", "mockito-junit-jupiter"]
 log4j = ["log4j-api", "log4j-core", "log4j-slf4j-impl", "log4j-slf4j2-impl", "log4j-web"]
+micronaut-annotation-processor = ["picocli-codegen", "micronaut-inject"]
+micronaut-cli = ["picocli", "micronaut-runtime", "micronaut-picocli", "micronaut-inject"]
+micronaut-test = ["micronaut-test-core", "micronaut-test-junit5"]
 slf4j = ["jul-to-slf4j", "jcl-over-slf4j", "log4j-over-slf4j"]
 temporal = ["temporal-sdk", "temporal-serviceclient"]
 
 [plugins]
+kapt = { id = "org.jetbrains.kotlin.kapt", version.ref = "kotlin" }
```

Comment:
We've decided to deprecate the usage `deps.toml` entirely and instead hard-code dependency versions everywhere. Right now it's just too easy for unintended dependency changes to sneak in unnoticed.

Maintaining such a file may have made sense in the days of the monorepo and pre-CDK but at this point it has more downsides than upsides.

Reply:
> We've decided to deprecate the usage `deps.toml` entirely and instead hard-code dependency versions everywhere. Right now it's just too easy for unintended dependency changes to sneak in unnoticed.
> 
> Maintaining such a file may have made sense in the days of the monorepo and pre-CDK but at this point it has more downsides than upsides.

@postamar Ugh...ok.  I don't agree with this decision but if you want me to copy pasta deps all over the place, I will make that change.